src/flavors/snohomish/config.json

```diff
@@ -594,11 +594,11 @@
             ],
             "content": [
               {
-                "label": "Practice rotational grazing, never grazing grass below 3” deep",
```

Comment:
The ASCII changes seem fine except for these two-- the unicode typographer's quotes should be replaced with bunny ears (i.e. `"`).

Reply:
Or, alternatively, just write out the phrase: `...below three feet deep`.

Reply:
Ah, nice catch. I'll update it by spelling it out with `...below three inches deep`.